src/modules/dashboard/components/ActionsPage/StakingWidget/StakingSlider.tsx

```diff
@@ -138,10 +166,29 @@ const StakingSlider = ({
           max={100}
           disabled={!canUserStake}
           appearance={appearance}
-          exceedLimit={exceedLimit}
+          handleLimitExceeded={onLimitExceeded}
         />
       </div>
-      {showError && <StakingValidationError stakeType={errorStakeType} />}
+      {errorStakeType && (
```

Comment:
After I fully stake a motion, I'm seeing this error: 
![Screenshot from 2021-09-15 14-52-06](https://user-images.githubusercontent.com/1193222/133428430-88217f90-8911-426b-8946-80371cdb3819.png)

This is not necessary something that you added, but since you're working on this, might as well take a look.

![Screenshot from 2021-09-15 14-51-41](https://user-images.githubusercontent.com/1193222/133428425-c29f6964-fc42-47fa-a399-60e8c565a4da.png)